### ISSUE TITLE
Does not seem to work with latest `chromium-browser` package on Raspbian 11

### ISSUE THREAD

**Issue by markphelps** [samsonmking/epaper.js#73]
**ejs version:** 2.1.1
**rpi-7in5-v2 version**: 2.0.3

After upgrading `chromium-browser` on Raspbian 11 `ejs` no longer seems to be able to launch chromium.

It just hangs at this stage and never renders anything:

## With Chromium 98

```
pi@pizero:~ $ chromium-browser --version
Chromium 98.0.4758.106 Built on Raspbian , running on Raspbian 11
```

```
pi@pizero:~ $ ejs refresh --debug --screenshotDelay=1000 rpi-7in5-v2 https://xyz.dev/
Connecting to rpi-7in5-v2 screen
/***********************************/
Current environment: Raspbian
Write and read /dev/spidev0.0
/***********************************/
```

After downgrading to the previous package version on Raspbian 11 it works again.

## With Chromium 95

```
pi@pizero:~ $ chromium-browser --version
Chromium 95.0.4638.78 Built on Raspbian , running on Raspbian 11
```

```
pi@pizero:/tmp $ ejs refresh --debug --screenshotDelay=1000 rpi-7in5-v2 https://xyz.dev/
Connecting to rpi-7in5-v2 screen
/***********************************/
Current environment: Raspbian
Write and read /dev/spidev0.0
/***********************************/
Connected
[DEBUG] Waiting an additional 1000ms before taking screenshot
[DEBUG] Screenshot delay complete
Waking up display
Displaying https://xyz.dev/
```

**Comment by KodeMunkie** [comment]
Any eyes and progress on this please?

**Comment by VictorioBerra** [comment]
Just a guess here, is it because this project depends on pupeteer-core and its way out of date? https://pptr.dev/chromium-support

https://github.com/samsonmking/epaper.js/blob/main/packages/core/package.json#L31

**Comment by samsonmking** [comment]
@markphelps @KodeMunkie @VictorioBerra apologies for the long delay :slightly_frowning_face:. As you're probably aware, this project uses puppeteer to render the images of the webpage. Each version of puppeteer supports a specific Chromium version. I've updated puppeteer to support the latest version of Chromium available on Raspberry Pi OS Bullseye: `104.0.5112.105`. Please let me know if you have any more issues!
